### ISSUE TITLE
WooCommerce stock management work incorrectly with radium class included

### ISSUE THREAD

**Issue by dedalx** [FrankM1/radium-one-click-demo-install#18]
This looks very strange but with this one click demo install class included woocommerce stock management for products start works incorrectly. I am using WP 4.2.1 and latest WooCommerce version.

How to represent issue:

1) Add radium class to theme
2) Install WooCommerce and add some test product with Quantity for example 10 and check Manage stock for this item and set item as In stock
3) Add this item to cart in frontend (for example 1 item) and checkout with this product
4) Check product quanitity - you will see that it changed to -1 and Out of stock status (but should be 9). If you check order log you will see message from woocommerce - product stock changed from 0 to -1 (but should be from 10 to 9).
Now comment code line that include radium class and checkout with the same product again (but first set it In stock and change quantity back to 10 for example) - you will see that product stock goes to 9 correctly.

**Comment by Ansif** [comment]
Yes, exactly i'm also facing this issue. My client said, he has that problem. But after a long time research i found WooCommerce issue is from Radium One Click Install. And this looks really strange. Even not able to predict this issue (WooCommerce to One Click Install).

I was, trying to get where that issue is from each and every files in my theme that is related to WooCommerce files for 10+ hours.

Did you fixed this issue? Or any solution for this?
